content/ko/docs/concepts/overview/working-with-objects/names.md

```diff
@@ -24,10 +24,20 @@ weight: 30
 
 {{< glossary_definition term_id="name" length="all" >}}
 
+**이름은 동일한 리소스의 모든 [API 버전](/docs/concepts/overview/kubernetes-api/#api-groups-and-versioning)에서 
+고유해야 한다. API 리소스는 API 그룹, 리소스 유형, 
+네임스페이스(네임스페이스 범위 리소스의 경우), 그리고 이름으로 구분된다. 즉, 이 맥락에서 API 버전은 고려 대상이 아니다.**
+
 {{< note >}}
 물리적 호스트를 나타내는 노드와 같이 오브젝트가 물리적 엔티티를 나타내는 경우, 노드를 삭제한 후 다시 생성하지 않은 채 동일한 이름으로 호스트를 다시 생성하면, 쿠버네티스는 새 호스트를 불일치로 이어질 수 있는 이전 호스트로 취급한다.
 {{< /note >}}
 
+리소스 생성 요청에 `generateName`이 `name` 대신에 제공되면 서버가 이름을 생성한다.
+`generateName`을 사용하면, 제공된 값은 이름의 접두사로 사용되고, 서버는 생성된 접미사를 여기에 
+붙인다. 이름이 생성되더라도 기존 이름과 충돌하여 HTTP 409 응답이 발생할 수 있다. 이는
+쿠버네티스 v1.31 이상의 버전에서 서버가 HTTP 409 응답을 반환하기 전에 고유한 이름을 
+최대 8번까지 생성하려고 시도하므로 이러한 문제 발생할 가능성이 훨씬 낮아졌다.
```

Comment:
```suggestion
쿠버네티스 v1.31 이상에서는 서버가 HTTP 409 응답을 반환하기 전에 최대 8회까지
고유한 이름을 생성하려고 시도하므로 이러한 일이 발생할 가능성이 훨씬 낮아졌다.
```

> This became far less likely to happen in Kubernetes v1.31 and later, since the server will make up to 8 attempt to generate a unique name before returning a HTTP 409 response.

원문의 느낌을 살려 자연스럽게 다듬어 보았습니다.